.travis.yml

```diff
@@ -1,3 +1,8 @@
+sudo: required
```

Comment:
Also, @justinpotts has finished his internship. I'm sure he's keen to see this land, but he might be a little busy over the next few weeks. I'm happy to take over if there are additional changes needed.